Header header
int32 image_height
int32 image_width
int32 object_area
bool object_detected
int32 object_x
int32 object_y
